src/persistance/fs-utils/index.ts

```diff
@@ -35,8 +35,31 @@ export const readDirIfExists = async (dirName: string): Promise<string[]> => {
   return fs.readdir(dirName);
 };
 
+export const readFileIfExists = async (location: string): Promise<unknown> => {
+  try {
+    const content = await fs.readFile(location, "utf-8");
+    return Promise.resolve(content);
+  } catch (e) {
+    return Promise.reject({});
```

Comment:
Don't catch all errors and return empty object. Only return empty object for the case that the file doesn't exist. For all other cases, the error should not be consumed. It should bubble up. Take example for the `readDirIfExists` function.